src/app/views/home/sections/Landing.scss

```diff
@@ -70,10 +70,25 @@
 			.hero-wrapper {
 				display: flex;
 				flex-direction: column;
-				justify-content: flex-start;
+				justify-content: center;
 				z-index: 3;
 			}
 
+			.jeep-wrapper {
```

Comment:
There are some sizing issues on narrower views: we need the jeep to fit on the screen.

Reply:
I had to modify some global variables in [here](https://github.com/HackAtUCI/ZotHacks-Site/blob/feature/landing-jeep/src/globals/zothacks-styles.scss#L11-L13) in order to better accommodate for the jeep resizing. Let me know if there is any other concerns with the jeep size.

Reply:
Changing the breakpoint values might affect the layout in other places, although I was thinking of redoing the container sizing, so for now, it might be okay. There are still some image scaling issues on Safari, seems like the image is stretched in one dimension but not the other.

Reply:
I fixed the issue on Firefox, can you check and confirm if the issue has also been fixed on Safari?

Reply:
I reworked most of the layout and positioning. Can you confirm things look alright?

Reply:
It looks good for mobile phones. However, for screens with larger widths like my laptop and the iPad, there are white bars on the sides. Is that intentional?

Reply:
We'll be replacing the background anyway (Riley is working on #51), so you can ignore that for now.